src/logger/loggerTaskEx.c

```diff
@@ -246,8 +247,7 @@ void loggerTaskEx(void *params)
                 }
 
                 /* send the sample on to the telemetry task(s) */
-                if (sampledRate >= telemetrySampleRate ||
-                    currentTicks % telemetrySampleRate == 0)
+                if (!isHigherSampleRate(sampledRate, telemetrySampleRate))
```

Comment:
we need this line: currentTicks % telemetrySampleRate == 0

otherwise if they have a config that coincidentally does not include the telemetry sample rate we may not queue any telemetry records.  Did you test this for regressions under varying conditions?